pkg/api/validation.go

```diff
@@ -0,0 +1,8 @@
+package api
+
+func ValidateRange(period int32) bool {
+	if period == 7 || period == 30 || period == 90 {
+		return true
+	}
+	return false
```

Comment:
refactor (non-blocking): No need for an explicit return of true or false unless this a preferred pattern in Go (I'm not a Gopher 😅)

```suggestion
	return period == 7 || period == 30 || period == 90
```

Reply:
Not a pattern, might be a side effect from too much ```if err != nil``` 😂

Reply:
```go
return period == 7 || period == 30 || period == 90
```

is valid Go and it really comes down to personal preference: there will be people who say this is a prime use case for a `switch` in Go in order to reduce the boolean mind games someone reading your code might need to do to figure out what's going on. Like:

```go
switch range {
case 7:
	return true
case 30:
	return true
case 90:
	return true
default:
	// invalid range
	return false
}
```

but others would say that's too verbose and ends up being crufty 🤷🏼 

Anywho, I'm good with what's written here and makes sense to me!

Reply:
> there will be people who say this is a prime use case for a switch in Go in order to reduce the boolean mind games

true, there is also this way with the switch.
```go
switch range {
case 7,30,90:
	return true
default:
	// invalid range
	return false
}
```
There is always the even-odd check meme way of doing it as well xD